contents/blog/what-to-ask-in-interviews.md

```diff
@@ -0,0 +1,114 @@
+
+Early stage founders are (or should be) super-mega-extra-desperately keen to have product market fit - it's all that really matters. The ones that will succeed are those that are honest about this (or those that have it already) and are prioritizing it. Many will think or say (intentionally or through self-delusion) that they have it when they don't. Low user or revenue numbers and vague answers to the example questions above are a sign that it isn't there. Product market fit [is _very_ obvious](https://www.ycombinator.com/library/5z-the-real-product-market-fit).
+
+Avoid:
+
+* Founders who aren't spending a ton of time talking and selling to users pre-product-market fit - it's a flag to attempt to outsource/delegate this
+* Companies where you don't understand the product or its benefits - if it can't be explained to you, others will struggle
+* Solutions looking for a problem
+
+If there is no product-market fit, but you think the team could find it, it's very important the company has lots of runway (it'll be hard to raise more money without product-market fit, so time to find it is crucial), that the startup doesn't have a huge team (it's harder to make rapid changes in direction with a lot of people), and that the company is fast (look out for bureaucracy, management, or a team that can't work well with ambiguity).
+
+## How much runway does the company have? Does their spending look within reason?
+
+"Are you [default alive](http://www.paulgraham.com/aord.html)?"
+
+This means that the company whilst not profitable, is growing fast enough that it will become profitable ahead of running out of money.
+
+Avoid:
+
+* Companies that don't know this or don't care about it, given [the current climate](https://techcrunch.com/2022/05/06/the-venture-slowdown-isnt-coming-its-here/) - this may change in an easier fundraising environment for early stage, capital-intense business models
+* Companies that you don't believe could get to default alive due to a lack of time and speed to get stuff done, if they aren't
+
+There are exceptions. For example, very low revenue numbers may lead to unrealistic growth rates - if I went from $1 to $2 of revenue yesterday, we'd be pretty darn rich if we kept that growth rate up.
+
+"What's your runway? What assumptions are you making to calculate this?"
+
+If a company doesn't have product-market fit, spending should be minimal so there's as much time as possible to find it. Spending a lot and having a big team just makes it harder to change direction rapidly.
+
+If a company has strong product market fit _and_ has a ton of traction, a shorter runway is ok.
+
+Avoid: 
+
+* Companies that assume wild increases in revenue to survive, within a short timeframe
+
+## What's the culture like?
+
+"What are the company values, and why? Can you give me some specific ways you've followed them?"
+"Who decides what to build?"
+"What do you look for when you hire?"
```

Comment:
I personally hate getting this question, as it should be obvious in the job spec. If you need to ask the question, the company has done a bad job communicating this. 

A more interesting question could be 'What are the things or behaviors that would cause a person in this role to fail?'

Reply:
I would also ask how a typical work day looks like at the company

Reply:
+1 to both of these suggestions.

Reply:
added the above!